README.md

```diff
@@ -1,14 +1,31 @@
 # Knative Serving Operator
 
+Knative Serving Operator is a project aiming to deploy and manage Knative Serving in an automated way.
+
 The following will install [Knative Serving](https://github.com/knative/serving)
 and configure it appropriately for your cluster in the `knative-serving`
 namespace:
 
 ```
 kubectl apply -f deploy/crds/serving_v1alpha1_knativeserving_crd.yaml
+```
+
+To install the operator from the source code, run the command:
+
+```
+ko apply -f deploy/
```

Comment:
Should we switch to using the config dir similar to other repos? Maybe in a follow on patch?

Reply:
That is how I initially changed in this PR, but it seems we can keep the change as little as possible, by reusing dir deploy.

Reply:
would it be sufficient to have a symlink there for the time being?

my 2c. As this is blocking the setting up of prow, I am leaning towards getting this in first and cleaning it up later,

Reply:
We can come back to the naming convention later.

Reply:
A symlink is fine with me, and honestly, I don't want to be the only one arguing to keep operator-sdk cruft around if operatorhub.io compliance isn't in the working group's scope. I'm happy to accept the majority decision on that and any other naming convention changes.